src/shared/components/Partner/styles.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+
+export const PartnerWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
let's try to keep our styles as clean as possible and use bootstrap classes instead if possible,

e.g `d-flex align-items-center`

Reply:
Thank you for your review. 
Fixed at https://github.com/MecenatOrg/frontend/pull/7/commits/07f455e542f0c981a1b2a472dc1f9eb49e495e6c

Reply:
I return old code because @lucypoly  say we mast don't use className